symplyphysics/laws/condensed_matter/current_density_from_mobility.py

```diff
@@ -42,6 +46,27 @@
     (-electrons_concentration * electrons_mobility + holes_concentration * holes_mobility) *
     electric_intensity)
 
+## This law might be derived via law for current density in metals.
+
+density_velocity_law_electrons = density_velocity_law.law.subs({
+    density_velocity_law.charge: charge,
+    density_velocity_law.charge_carriers_concentration: -electrons_concentration,
+    density_velocity_law.drift_velocity: electrons_mobility * electric_intensity,
```

Comment:
Do we have a law for it? If not, either write a TODO to fix it, or make a corresponding law.